CommaTheBot/CommaTheBot.py

```diff
@@ -0,0 +1,68 @@
+# example invocation
+# python CommaTheBot.py --file filtered_ol_dump_2022-06-06.txt.gz --limit 1
+
+from olclient.bots import AbstractBotJob
+import copy
+import gzip
+import re
+
+class CommaTheBotJob(AbstractBotJob):
+    pattern = None
+
+    def __init__(self):
+        super().__init__()
+        articles = [
+            '[Tt]he',
+            '[Dd]er', '[Dd]ie', '[Dd]as',
+            '[Ll]e', '[Ll]a', '[Ee]l',
+            '[Ll]os', '[Ll]as', '[Ll]es',
+        ]
+
+        self.pattern = re.compile(rf"^([\w ,]*), ?({'|'.join(articles)})$")
+
+    def needs_fixing(self, edition_title: str) -> bool:
+        if edition_title == None: return False # no title given
```

Comment:
See PEP8 discussion about comparing to ˋNone`.

Reply:
Addressed in 0a67693, thank you.